spec/services/shopify_import/products/create_spec.rb

```diff
@@ -57,11 +57,28 @@
         end
       end
 
+      context 'spree variants' do
+        let(:spree_product) { Spree::Product.find_by!(name: shopify_product.title) }
+
+        it 'creates spree variant' do
+          expect { subject.save! }.to change(Spree::Variant, :count).by(2)
+        end
+
+        it 'assings variants to product' do
+          subject.save!
+          expect(Spree::Variant.last.product).to eq spree_product
```

Comment:
I really don't understand the need to have everything as a `let` somewhere outside of the `it` block. The code should be written for readability. Currently the scenario for reading the code looks like this:

1. Read `subject.save!`
2. What's the `subject`. Let me check. (scroll to the top of the file)
3. Scroll back down: `expect(Spree::Variant.last.product).to eq spree_product`
4. What's `spree_product`. Scroll up. (hopefully not missing the exact `spree_product` declaration for some other: https://github.com/spark-solutions/spree_shopify_importer/blob/b7a76a31f52c52bed9e2d777a3c74d688da45ed7/spec/services/shopify_import/products/create_spec.rb#L26

I usually prefer writing specs "inline":

```ruby
it 'assings variants to product' do
  # arrange
  shopify_product = ShopifyAPI::Product.find(9_884_552_707)
  product_data_feed = create(:shopify_data_feed,
                             shopify_object_id: shopify_product.id, data_feed: shopify_product.to_json)

  # act
  ShopifyImport::Products::Create.new(product_data_feed).save!
  expected_product = Spree::Product.find_by!(name: shopify_product.title)

  # assert
  expect(Spree::Variant.last.product).to eq(expected_product)
end
```

I add couple of extra lines of code, but debugging failing specs is so much easier.